RWKV-v4neo/src/model.py

```diff
@@ -535,21 +534,15 @@ def forward(self, idx):
         x_emb = x
 
         if args.tiny_att_dim > 0:
-            for block in self.blocks:
-                if args.grad_cp == 1:
-                    if args.lora:
-                        x = torch_checkpoint(block, x, x_emb, use_reentrant=False)
-                    else:
-                        x = deepspeed.checkpointing.checkpoint(block, x, x_emb)
+            for i, block in enumerate(self.blocks):
+                if i and args.grad_cp == 1:
```

Comment:
Do you mean that `--grad_cp 1` works lora and deepspeed on your side? It didn't work for many others as well as myself, and I've looked into the implementation and confirmed it's the `use_reentrant` switch that matters. How did you fix that?